floris/tools/optimization/legacy/scipy/yaw_wind_rose.py

```diff
@@ -243,7 +243,7 @@ def _get_initial_farm_power(self):
                     self.fi.reinitialize_flow_field(
                         wind_direction=[self.wd[i]],
                         wind_speed=[self.ws[i]],
-                        turbulence_intensity=self.ti[i],
+                        turbulence_intensities=[self.ti[i]],
```

Comment:
I'm not sure what the optimization/legacy/scipy/ package refers to---do these currently run with v3, and are they expected to run with v4? If not the latter (not expected to run with v4), I'd be in favor of returning these to `turbulence_intensity` so that it does not appear to be updated to v4 compatibility.

In that case, I'd just revert all files in optimization/legacy/scipy.

Reply:
that works for me, or delete?  The legacy could, if it's for v3, can just remain attached to v3 code?  Do we need a legacy folder I guess?

Reply:
It can probably be deleted, but let's save that for a separate PR, so I'd say just change them back for this one.

Reply:
agree changing now